layers/core_checks/cc_cmd_buffer_dynamic.cpp

```diff
@@ -424,6 +424,19 @@ bool CoreChecks::ValidateDrawDynamicStatePipeline(const LAST_BOUND_STATE& last_b
             }
         }
     }
+    if (pipeline.IsDynamic(VK_DYNAMIC_STATE_SAMPLE_LOCATIONS_EXT) &&
+        !pipeline.IsDynamic(VK_DYNAMIC_STATE_RASTERIZATION_SAMPLES_EXT) &&
+        cb_state.dynamic_state_status.cb[CB_DYNAMIC_STATE_SAMPLE_LOCATIONS_EXT]) {
+        if (cb_state.dynamic_state_value.sampleLocationsInfo.sampleLocationsPerPixel !=
+            pipeline.MultisampleState()->rasterizationSamples) {
+            const LogObjectList objlist(cb_state.commandBuffer(), pipeline.pipeline());
+            skip |= LogError(vuid.sample_locations_07482, objlist, loc,
+                             "sampleLocationsPerPixel set with vkCmdSetSampleLocationsEXT() was %s, but "
+                             "VkPipelineMultisampleStateCreateInfo::rasterizationSamples pipeline was created was is %s.",
```

Comment:
english here is a little off

```suggestion
                             "VkPipelineMultisampleStateCreateInfo::rasterizationSamples from the pipeline was %s.",
```